File: app/src/main/java/com/v/designmode/MyApplication.java
package com.v.designmode;

import android.app.Application;

import com.v.network_architecture.HttpHelper;
import com.v.network_architecture.xutils.XutilsProcessor;

public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

//        HttpHelper.init(new VolleyProcessor(this));
//        HttpHelper.init(new OkHttpProcessor());
        HttpHelper.init(new XutilsProcessor(this));
    }
}

File: kotlin/src/main/java/com/v/kotlin/MyApplication.kt
package com.v.kotlin

import android.app.Application
import android.content.Context
import com.v.kotlin.database.AppDatabase

class MyApplication : Application() {
    companion object{
        private lateinit var mApplication: Application
        fun getContext(): Context {
            return mApplication
        }
    }

    override fun onCreate() {
        super.onCreate()
        mApplication = this
        AppDatabase.initApplication(this)
    }
}